Dockerfile

```diff
@@ -8,16 +39,9 @@ LABEL org.opencontainers.image.documentation="https://github.com/swissgrc/docker
 # Make sure to fail due to an error at any stage in shell pipes
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-# Prerequisites
-
-# renovate: datasource=repology depName=debian_12/ca-certificates versioning=loose
-ENV CACERTIFICATES_VERSION=20230311
-
-# Ca-Certificates is required for connection to Azure DevOps
-RUN apt-get update -y && \
-  apt-get install -y --no-install-recommends ca-certificates=${CACERTIFICATES_VERSION} && \
-  apt-get clean && \
-  rm -rf /var/lib/apt/lists/*
+WORKDIR /
+# Copy Flux CLI from build stage
+COPY --from=build /usr/local/bin/flux /usr/local/bin/flux
```

Comment:
Can you add a run command afterwards calling the Flux CLI (e.g. output its version) to test if the tool is actually working (like e.g. done here: https://github.com/swissgrc/docker-azure-pipelines-git/blob/develop/Dockerfile#L102)